docs/guide/topics/error_handling.rst

```diff
@@ -0,0 +1,361 @@
+            print(f"Ignoring unhandled exception in command {ctx.command!r}")
+            traceback.print_exception(type(error), error, error.__traceback__)
+            
+
+.. note::
+    We don't want our global error handler to 'eat' our unhandled errors without displaying anything in the console, so we add
+    an :ref:`else <py:else>` statement at the end of our logic and handle all the remaining 'unhandled' errors.
+
+Local Error Handlers
+~~~~~~~~~~~~~~~~~~~~
+
+In :class:`Cogs <~ext.commands.Cog>`
+
+***************************************
+
+To handle errors in only a single :class:`~ext.commands.Cog` you override it's :meth:`~ext.commands.Cog.cog_command_error` method,
+which gets called when an error occurs inside of this Cog.
+
+**Example**
+
+.. code-block:: python3
+    :emphasize-lines: 8
+
+    class ExampleCog(commands.Cog):
+
+        async def cog_check(ctx):
+            """ Custom check for this cog only. """
+            if ctx.guild and ctx.guild.id != 336642139381301249:
+                raise commands.DisabledCommand
+
+        async def cog_command_error(self, ctx, error):
+            """ This error handler is only for this specific Cog. """
+
+            if isinstance(error, commands.DisabledCommand):
+                await ctx.send(f"The command {ctx.command!r} cannot be used in this server!")
+      
+In a :class:`~.ext.commands.Command`
+
+************************************
+
+
+To handle errors for a specific command, we decorate the function with :meth:`~ext.commands.Command.error`, which will 
+only get called for that specific command.
+
+You can use the same function for multiple commands too, simply by adding multiple decorators on top of it.
+
+**Example**
+
+.. code-block:: python3
+    :emphasize-lines: 17, 18
+
+    @bot.command()
+    @commands.has_permissions(kick_members=True)
+    @commands.bot_has_permissions(kick_members=True)
+    async def kick(ctx, member: discord.Member):
+        """ Kicks a member from this server. """
+        await member.kick()
+        await ctx.send(f"Kicked {member}! Bye bye.")
+
+    @bot.command()
+    @commands.has_permissions(ban_members=True)
+    @commands.bot_has_permissions(ban_members=True)
+    async def ban(ctx, member: discord.User):
+        """ Bans a member from this server. """
+        await ctx.guild.ban(user)
+        await ctx.send(f"Banned {member}! Begone.")
+
+    @kick.error
+    @ban.error
+    async def kick_error(ctx, error):
+        """ This error handler is only for the `kick` and `ban` commands """
+
+        if isinstance(error, commands.MissingPermissions):
+            formatted_perms = ', '.join(error.missing_perms)
+            await ctx.send(f'You are missing the following permissions: {perms}')
+
+        elif isinstance(error, commands.BotMissingPermissions):
+            formatted_perms = ', '.join(error.missing_perms)
+            await ctx.send(f'I am missing the following permissions: {perms}')
+
+
+.. warning::
```

Comment:
I think a warning is a bit much. A reference to the "using global and local error handlers together" section might also be worthwhile. 
I also disagree with the line 
> If you're handling unhandled errors 
    in it, do not handle them in local error handlers, as this may cause them to be handled twice or more.

There are a number of ways to handle the same error in different ways in different handlers, which you cover later.